src/99-appendices/08-coordinate-systems.md

```diff
@@ -151,6 +151,19 @@ Restricted keywords for the `<CoordSysType>CoordinateSystem` field in the
     [ACPC site](https://www.fieldtriptoolbox.org/faq/acpc/) on the FieldTrip
     toolbox wiki.
 
+-   `fsnative`: The subject-specific FreeSurfer T1w space. This corresponds to
+    the original scanner image after it is re-sampled and re-scaled
+    to an isotropic 1 mm 256x256x256 volume in RAS orientation. The origin
+    is wherever the original origin of the ``orig.mgz`` file is. One SHOULD
+    align the origin to the ``ACPC`` before using this coordinate system.
```

Comment:
what is `orig.mgz`? 

>  One SHOULD align the origin to the ``ACPC`` before using this coordinate system

why SHOULD one do that?  ... and why not MUST?

Reply:
I suppose it can be a MUST, so that way fsnative origin is assumed to be AC of subjects brain but I think this might depart from what ppl consider fsnative generally. I don't think acpc alignment is required by all ppl? 

Cc: @robertoostenveld

Reply:
if it is not AC, but a random other location in the body (e.g. the left big toe), then I would say that it is not `fsnative`. In that case it should be specified with `other` and the details are to be specified in the CoordinateSystemDescription.